### ISSUE TITLE
Show the window from the desktop where the mouse is located

### ISSUE THREAD

**Issue by chenxiankong** [lwouis/alt-tab-macos#3354]
**Is your feature suggestion related to a problem? Please describe.**
No

**Describe the solution you'd like**
Show the window from the desktop where the mouse is located
Currently there are two options: visible desktop and all desktops. I hope to add an option for the desktop where the mouse is located. The content of 
 should be consistent with the window displayed by the Apple MissionControl of the desktop.
Thank You

best wishes for you

**Comment by lwouis** [comment]
Hi,

I think you can achieve your goal with these preferences :

<img width="586" alt="image" src="https://github.com/lwouis/alt-tab-macos/assets/106195/1154d8b6-fba6-4abd-99af-1b67f13e4af5">

then

<img width="586" alt="image" src="https://github.com/lwouis/alt-tab-macos/assets/106195/75bfe9aa-bde7-45de-b03b-94ed39336377">

Thank you 🙇

**Comment by chenxiankong** [comment]
From the description, this function meets my needs, but the actual content does not quite meet my needs. 
 Operation: Assign application A to all desktops in the dock. 
 Current performance: It will appear on any desktop using altTab .
Expected performance: Remain consistent with the contents of the missionControl, only appearing on the last desktop it was on

Thank You
Best wishes for you ~

**Comment by lwouis** [comment]
Hi @chenxiankong,

I'm not sure I understand what you're saying. Could you please share a video showcasing the behavior you see from AltTab, and the behavior of Mission Control which you would like to see?

Thank you 🙇

**Comment by chenxiankong** [comment]
Sorry, for some reasons, I can't show you the video. But I will try to describe it clearly below. 

First, connect an external display, so we now have two desktops: M1, M2
Secondly, open 3 Application: A1, A2, A3, assign A1 to all desktops. 
The last time A1 was in M1, A2 was also in M1, and A3 was in M2

Open the missionControl of macos, you can see the windows of A1 and A2 in M1 (Please note that even if A1 is assigned to all desktops, it will only appear in the desktop of one display in missionControl.), A3's window settings in M2.

altTab are consistent with the settings you gave .

1. Start the operation: First click the mouse on M1, and then open alttab. You can see the windows of A1 and A2. This is normal. 
2. Then move the mouse directly to M2 (without clicking) and open alttab. , only the window of A3 can be seen at this time. (This is in line with my needs) 
3. Finally, click the mouse on M2, and then open altTab. At this time, you can see the windows of A1 and A3 (this is not in compliance with Expected).
To sum up, whether the mouse clicks on the current desktop affects the behavior of alttab. I don’t quite understand the logic.

**Comment by lwouis** [comment]
@chenxiankong thank you for your message

You're assigning the app to all Desktops? You used this UI in the Dock to assign it?

<img width="418" alt="image" src="https://github.com/lwouis/alt-tab-macos/assets/106195/133357a6-82ba-4d66-9742-18ef415cd784">

If you do that, AltTab will show this app's windows on every Desktop. This makes sense, because the app is indeed on every Desktop. If you move between Desktops, the app's windows will follow you.

I'm not understanding the issue, if there is one.

Could you please try and share a video?

**Comment by lwouis** [comment]
Hi @chenxiankong,
Could you please reply? Otherwise, I will have to close the ticket.
Thank you 🙇

**Comment by slkmikhail** [comment]
.
